pennylane/templates/subroutines/mod_exp.py

```diff
@@ -84,11 +86,14 @@ def __init__(
 
         output_wires = qml.wires.Wires(output_wires)
 
+        if work_wires is None:
+            raise ValueError("Work wires must be specified for ModExp")
```

Comment:
Maybe excessive? Mostly just thinking that it's an allowed option for many of the other quantum arithmetic templates, so it's not an unlikely thing to do.